Debug App/Sources/View Controllers/MerchantSessionAndSettingsViewController.swift

```diff
@@ -447,7 +447,9 @@ class MerchantSessionAndSettingsViewController: UIViewController {
                 applePayOptions: PrimerApplePayOptions(
                     merchantIdentifier: "merchant.checkout.team",
                     merchantName: merchantNameTextField.text ?? "Primer Merchant",
-                    isCaptureBillingAddressEnabled: false)),
+                    isCaptureBillingAddressEnabled: false,
+                    showApplePayForUnsupportedDevice: false,
+                    checkProvidedNetworks: false)),
```

Comment:
I think this needs to specify it's for apple pay, e.g. `checkProvidedNetworksAvailableForApplePay` (or something like that)